apps/src/templates/SchoolDataInputs.jsx

```diff
@@ -0,0 +1,141 @@
+import PropTypes from 'prop-types';
```

Comment:
Worth noting: this file is a WIP. I initially converted the functionality of the old component while maintaining the validation. I have not yet fixed all the styling issues, and there are still some pieces I need to build out. Is it better to check this in or not merge it at all until it's done? Again, not actually in use yet.

Reply:
maybe asked a different way -- what's the benefit of merging this now? Early PR feedback, simpler PRs in the future, something else? (This would also help reviewers know what to focus on)

Reply:
Very helpful question- yes I was hoping for simpler PR's instead of just opening one up that's a ton of changes and validation at once. I'll update the description and resolve this